api/core.py

```diff
@@ -370,7 +202,7 @@ def _archivestream(self, ticket):
         stream.close()
 
     def download(self):
```

Comment:
Suggest rename to bulk_download to distinguish between downloading a tar and downloading an individual file